config/config.go

```diff
@@ -40,3 +46,24 @@ func SetProjectID(_projectID string) {
 func GetProjectID() string {
 	return projectID
 }
+
+func SetLogLevel(level string) {
+	switch level {
+	case "DEBUG":
+		logLevel = DEBUG
+	case "INFO":
+		logLevel = INFO
+	case "WARN":
+		logLevel = WARN
+	case "ERROR":
+		logLevel = ERROR
+	case "CRITICAL":
+		logLevel = CRITICAL
+	case "ALERT":
+		logLevel = ALERT
+	}
```

Comment:
An error or warning is need to be displayed if all options do not match.

Reply:
Done.